onnxruntime/core/framework/ort_format_model_utils.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
nit: can we name the files with a more specific name?
e.g., static_kernel_def_hashes.h/cc